src/utils/transformationDescription.ts

```diff
@@ -84,9 +84,9 @@ export function useActiveTransformations(
               .transformerFunction.kind === "datasetCreator"
           ) {
             setErrMsg(
-              `Error updating ${
+              `Error updating "${
                 (description as DatasetCreatorDescription).output
```

Comment:
This `description.output` seems to be the context name. Is it worth making a phone request here to `getDataContext` and then extracting the `readableName`? This would address one of Flannery's complaints.

Reply:
Yeah I think that's a good idea.